modules/sor/sorV1/sorV1.service.ts

```diff
@@ -0,0 +1 @@
+// This will hold logic for querying SORV1 trade. Either via call to existing API or SORV1 package directly.
```

Comment:
Is it better to query the existing SOR API (which is SORV1) or bring that functionality into this backend? 
I guess there wouldn't be an issue with network delays so API call might be more efficient if the goal is to eventual remove SORV1 dependency.

Reply:
Not sure what you mean by SOR API? For the Beethoven website we have the SOR deployed in a Lambda, is that what you mean?

Reply:
I would say use the API call, but you'll need to add an exception for networks that don't support the balancer API (fantom/optimism). In those instances, just do nothing.

Reply:
or call our API :)